tests/plugin.py

```diff
@@ -15,8 +15,8 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption("randomly_seed") == 'default':
-        config.option.randomly_seed = 42
+   # if config.getoption("randomly_seed") == 'default':
```

Comment:
why is this commented out?

Reply:
The tests were failing with weird error `AtributeError Namespace object has no attribute randomly_seed